src/containers/DataGUI.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component, PropTypes } from 'react';
```

Comment:
[`fa89d19`](https://github.com/jekyll/jekyll-admin/pull/354/commits/fa89d196f9e8eca8c70aaf6ecb1a32fe13562f4a) to easily demonstrate what changed between `MetaFields` and `DataGUI`